stdlib/sql/integration_test.flux

```diff
@@ -0,0 +1,191 @@
+package integration_test
```

Comment:
```suggestion
package sql_test
```

Which should fix the CI failure you are seeing. There can only be one package and and test package per directory :shrug:

Reply:
Thank you @Marwes :pray: